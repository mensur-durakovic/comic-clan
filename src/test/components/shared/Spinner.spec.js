import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '../../../components/shared/Spinner/Spinner';

test('Spinner component', () => {
  const tree = renderer.create(<Spinner text="test"/>).toJSON()
  expect(tree).toMatchSnapshot()
})