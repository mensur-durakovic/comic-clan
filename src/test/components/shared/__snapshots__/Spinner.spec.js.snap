// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component 1`] = `
<div
  className="sc-AxiKw dLTsww"
>
  <span
    className="sc-AxirZ cmuFyj"
  />
  <svg
    className="sc-AxjAm gCQBpC"
    viewBox="0 0 50 50"
  >
    <circle
      className="path"
      cx="25"
      cy="25"
      fill="none"
      r="20"
      strokeWidth="5"
    />
  </svg>
</div>
`;
